addons/mail/static/src/utils/common/hooks.js

```diff
@@ -442,15 +443,28 @@ export function useSelection({ refName, model, preserveOnClickAwayPredicate = ()
 }
 
 export function useMessageEdition() {
+    /** @type {import("@mail/core/common/store_service").Store} */
```

Comment:
Note: better use `@type {import("models").Store}`, as this is not only shorter but also has any patches of `Store` in it too!